pkgs/tools/misc/uwuify/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, rustPlatform, libiconv }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "uwuify";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "Daniel-Liu-c0deb0t";
+    repo = "uwu";
+    rev = "v${version}";
+    sha256 = "sha256-tPmLqgrWi7wDoMjMrxodKp4S0ICwV9Kp7Pa151rHho0=";
+  };
+
+  cargoSha256 = "sha256-HUP6OEvoGJ/BtAl+yuGzqEp1bsxfGAh0UJtXz9/ZiK8=";
+  buildInputs = [ libiconv ];
+
+  meta = with lib; {
+    description = "Fastest text uwuifier in the West";
+    homepage = "https://github.com/Daniel-Liu-c0deb0t/uwuify";
```

https://github.com/Daniel-Liu-c0deb0t/uwu gives me a 404 error.

```suggestion
    homepage = "https://github.com/Daniel-Liu-c0deb0t/uwu";
```